src/yarp.c

```diff
@@ -12931,7 +12974,7 @@ yp_parser_free(yp_parser_t *parser) {
 // Parse the Ruby source associated with the given parser and return the tree.
 YP_EXPORTED_FUNCTION yp_node_t *
 yp_parse(yp_parser_t *parser) {
-    return parse_program(parser);
+  return parse_program(parser);
```

Comment:
I think this was probably unintentional?

Reply:
I was using emacs before (I do not write C very much) and I switched to an IDE.  I expected it would just indent.  I guess not.